helmfile.d/charts/grafana-dashboards/files/welcome.md

```diff
@@ -4,14 +4,12 @@
 
 Here you can find the most relevant features and changes for the last couple of releases of Compliant Kubernetes
 
+- Trivy Operator was upgraded to v0.20.1. **[v0.39]**
+- Velero was upgraded to v1.13.0. **[v0.39]**
+- Pods with the label `elastisys.io/prometheus-access: allow` can now access Prometheus. **[v0.39]**
```

Comment:
Not sure if we want to announce this or not since it's still required by a platform administrator to allowlist the namespaces where this is possible. Let me know what you think!

Reply:
```suggestion
- Pods can now be granted access to the API of Prometheus from Application Developer namespaces per request. **[v0.39]**
```
I think we just reword it, but as it is a new feature I do think we should announce it.